### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by peterboro**
In this pull request I implemented the following:

- Set up a new project with webpack that is based on the webpack exercise you have already completed.
- Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (<div> or <ul> element). The index.html file must be set as a template using the HTML Webpack Plugin.
- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
- description [string].
- completed [bool].
- index: [number].
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
- Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.
- All your source files (index.html, index.js and style.css) must be located in /src directory and your distribution files will be generated by webpack and served by webpack dev server from /dist folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
-# ToDoList
\ No newline at end of file
+# ToDoList
```
Comment (julie-ify) [inline README.md:1]: - [ ] You need to move all the files in the `webpack` folder to the root directory as the root directory is where the app start command should work from.

File: webpack/src/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  { description: 'Wash the dishes', isCompleted: false, index: 0 },
+  { description: 'Complete To Do List Project', isCompleted: false, index: 1 },
+];
+
+const todoList = () => {
+  let todoListContent = '';
+  todo.forEach((item) => {
+    todoListContent += `<li class="task"><input class="task-check" type="checkbox"><span class="list">${item.description}</span><i class='fa fa-ellipsis-v' style="float: right;"></i></li>`;
+  });
+  document.querySelector('.task-box').innerHTML = todoListContent;
```
Comment (julie-ify) [inline webpack/src/index.js]: - [ ] We can improve this design to be more similar to the template given. Kindly add some top and bottom padding to the tasks as we have in the template.

  Your Implementation | Template Design
  :---:|:---:
  one-1 | two-2

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @peterboro,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @peterboro , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by peterboro** [comment]
Thank you for the review.

**Pull request merged**